app/main/views.py

```diff
@@ -213,6 +213,23 @@ def notebooks():
         notebooks=notebooks,
         form=form)
 
+@main.route('/favorites', methods=['GET', 'POST'])
+@login_required
+def favorites():
+    if current_user.is_authenticated():
+        notes = Note.query.filter_by(
+            author_id=current_user.id,
+            is_deleted=False,
+            is_favorite=True).order_by(
+            Note.updated_date.desc()).all()
+        return render_template('app/app.html', notes=notes)
+    else:
```

Comment:
My bad, It's okey now ^^.

2015-12-15 17:52 GMT+01:00 Lev Lazinskiy <notifications@github.com>:

> In app/main/views.py
> <https://github.com/levlaz/braindump/pull/130#discussion_r47663652>:
>
> > @@ -213,6 +213,23 @@ def notebooks():
> >          notebooks=notebooks,
> >          form=form)
> >
> > +@main.route('/favorites', methods=['GET', 'POST'])
> > +@login_required
> > +def favorites():
> > +    if current_user.is_authenticated():
> > +        notes = Note.query.filter_by(
> > +            author_id=current_user.id,
> > +            is_deleted=False,
> > +            is_favorite=True).order_by(
> > +            Note.updated_date.desc()).all()
> > +        return render_template('app/app.html', notes=notes)
> > +    else:
>
> See note above, this is automatically done if the user is not logged in.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/levlaz/braindump/pull/130/files#r47663652>.
>